lambda/db/package.json

```diff
@@ -8,9 +8,7 @@
     "compile": "../node_modules/.bin/tsc && ../node_modules/.bin/tsc-alias",
     "build": "../node_modules/.bin/ncc build src/main.ts -o dist -m"
   },
-  "devDependencies": {
-    "sqlite3": "^5.0.8"
-  },
+  "devDependencies": {},
```

Comment:
Does this change the local dev experience?

Reply:
I ran the local dev env with out any issues after removing this, so should be fine